ns/namespace.go

```diff
@@ -238,3 +247,43 @@ func GetValueByNamespace(object interface{}, ns []string) interface{} {
 	}
 	return nil
 }
+
+func ReplaceNotAllowedCharsInNamespacePart(ns string) string {
+	//brakets
+	ns = strings.Replace(ns, "(", "_", -1)
+	ns = strings.Replace(ns, ")", "", -1)
+	ns = strings.Replace(ns, "[", "_", -1)
+	ns = strings.Replace(ns, "]", "", -1)
+	ns = strings.Replace(ns, "{", "_", -1)
+	ns = strings.Replace(ns, "}", "", -1)
+	//spaces
+	ns = strings.Replace(ns, " ", "", -1)
+	//punctuations
+	ns = strings.Replace(ns, ".", "_", -1)
+	ns = strings.Replace(ns, ",", "_", -1)
+	ns = strings.Replace(ns, ";", "_", -1)
+	ns = strings.Replace(ns, "!", "", -1)
+	ns = strings.Replace(ns, "?", "", -1)
+	//slashes
+	ns = strings.Replace(ns, "|", "_", -1)
+	ns = strings.Replace(ns, "\\", "_", -1)
+	ns = strings.Replace(ns, "/", "_", -1)
+	//carets
+	ns = strings.Replace(ns, "^", "_", -1)
+	//quotations
+	ns = strings.Replace(ns, "\"", "_", -1)
+	ns = strings.Replace(ns, "`", "_", -1)
+	ns = strings.Replace(ns, "'", "_", -1)
+	return ns
+}
```

Comment:
Consider loop which iterates over all not allowed character kept in notAllowedChars map